src/app/rpmostree-builtin-compose.c

```diff
@@ -32,6 +32,15 @@ static RpmOstreeCommand compose_subcommands[] = {
   { "tree", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
     "Install packages and commit the result to an OSTree repository",
     rpmostree_compose_builtin_tree },
+  { "install", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
+    "Install packages into a target path",
+    rpmostree_compose_builtin_install_root },
+  { "postprocess", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
+    "Perform final postprocessing on an installation root",
+    rpmostree_compose_builtin_postprocess_root },
+  { "commit", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
+    "Commit a target path",
```

Comment:
Can we make it consistent with the `tree` description and add ` to an OSTree repository`?